submissions/antonkottans/frogger/app.js

```diff
@@ -1,37 +1,35 @@
 class Field {
     constructor() {
-        this.maxreachableXCoord = 4;
-        this.maxreachableYCoord = 4;
         this.topIndent = 57;
-        this.SquareWidth = 101;
-        this.SquareHeight = 85;
+        this.squareWidth = 101;
+        this.squareHeight = 85;
+        this.maxreachableXCoord = 4 * this.squareWidth;
+        this.maxreachableYCoord = 4 * this.squareHeight;
     }
 }
 const field = new Field();
 
 class Enemy {
-    constructor({ x, y, speed, row }) {
+    constructor({ x, y, speed }) {
         this.x = x;
         this.y = y;
         this.speed = speed;
         this.sprite = "images/enemy-bug.png";
-        this.row = row;
     }
     checkCollision() {
-        const distanceOfCollision = 70;
-        const sameRow = () => player.y === this.row;
+        const distanceOfXCollision = field.squareWidth / 1.3;
```

Comment:
Magic numbers are [bad](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)